legacy_packages/sdk/src/evm/schema/contracts/custom.ts

```diff
@@ -171,7 +171,7 @@ export const CustomFactoryInput = /* @__PURE__ */ (() =>
  */
 export const ModularFactoryInput = /* @__PURE__ */ (() =>
   z.object({
-    hooksParamName: z.string(),
+    extensionsParamName: z.string(),
```

Comment:
The hook terminology was merged recently, but then it got changed to extension.

It could be breaking for some published contracts. I can keep both fields in that case.